src/Energon.php

```diff
@@ -44,26 +44,48 @@ public static function generate($prime = null)
     }
 
     /**
+     * Generates a set of numbers ready for use (with a custom max cap).
+     *
+     * @param int $max
+     *
+     * @return array
+     */
+    public static function generateMax($max)
```

Comment:
We could but then we'd always, when using generate and not using a predefined prime, have to pass `null, $max`. Are you sure?